### DISCUSSION TITLE
Retry when dns resolution fails?

### DISCUSSION THREAD

**Discussion by guyschlider** [aws/aws-sdk-js-v3#4426] [Q&A]
Hey
When some if our Lambda function fails to resolve dynamodb dns there are no retries made. Is this intentional?

{\"errno\":-3008,\"code\":\"ENOTFOUND\",\"syscall\":\"getaddrinfo\",\"hostname\":\"dynamodb.us-east-1.amazonaws.com\"

**Comment by yenfryherrerafeliz** [comment]
Hi @guyschlider, thanks for opening this discussion. Are you using the SDK for JavaScript when this error occurs?, if so, could you please provide a code snippet that can help us understand what exactly you are doing?

Note: Please redact any sensitive information.

Thanks!
